src/electionguard/facades/Hacl_Bignum4096.cpp

```diff
@@ -65,12 +116,32 @@ namespace hacl
 
     void Bignum4096::mod(uint64_t *a, uint64_t *res) const
     {
+#ifdef _WIN32
+        Hacl_Bignum4096_32_mod_precomp(context.get(), reinterpret_cast<uint32_t *>(a),
+                                       reinterpret_cast<uint32_t *>(res));
+#else
         Hacl_Bignum4096_mod_precomp(context.get(), a, res);
+#endif // _WIN32
     }
 
     void Bignum4096::modExp(uint64_t *a, uint32_t bBits, uint64_t *b, uint64_t *res,
                             bool useConstTime /* = true */) const
     {
+#ifdef _WIN32
+        if (bBits <= 0) {
```

Comment:
You can take out the if bBits <= 0 from both if and the else sections and just put it above once.  A function above does this already this way.

Reply:
Made this change